### ISSUE TITLE
package/region-provider does not bundle an es version, prevents use of rollup on client-dynamodb

### ISSUE THREAD

**Issue by antstanley** [aws/aws-sdk-js-v3#1243]
**Describe the bug**
I get the following Error

```
'defaultProvider' is not exported by node_modules/@aws-sdk/region-provider/build/index.js, imported by node_modules/@aws-sdk/client-dynamodb/dist/es/runtimeConfig.js
```
when using Rollup to bundle `@aws-sdk/client-dynamodb` 

**SDK version number**
`@aws-sdk/client-dynamodb v1.0.0-gamma.2`

**Is the issue in the browser/Node.js?**
Node.js

**Details of the browser/Node.js version**
Tried in these versions and all failed
v13.12.0
v12.18.0
v14.4.0

**To Reproduce (observed behavior)**
Bundle `@aws-sdk/client-dynamodb` using Rollup 
Will add a repo shortly

**Expected behavior**
No error on bundling

**Additional context**
I inspected the code for the `region-provider` package and noticed it only ships a commonjs module, and not the dual es and cjs modules like `@aws-sdk/client-dynamodb`. Due to the way exports work in es modules, my guess is that `region-provider` package does not export the `defaultProvider` method in a way compatible with es modules.

At a guess a resolution to this would be add the dual module system export to `region-provider`, so add the esmodule tsconfig file to export an esmodule version to `dist/es` and add  `module: dist/es/index.js`

**Comment by AllanFly120** [comment]
Hi @antstanley 

Thank you for reaching out. We are working on shipping  **all** the packages in both ES module and CJS module soon.

**Comment by antstanley** [comment]
Ok awesome! It looks like all packages don't export es module versions. So realise it's a big job to get around to all of it.

Happy to be a guinea pig, I bundle all my Lambda functions using Rollup. I typically exclude `aws-sdk` from the bundle because v2 is not bundle friendly, hoping for better things with v3, and it looks like it's going to be a lot easier to bundle with Rollup once it exports es modules for everything.

**Comment by trivikr** [comment]
ES version of @aws-sdk/region-provider was released with 1.0.0-gamma.3
https://github.com/aws/aws-sdk-js-v3/releases/tag/v1.0.0-gamma.3
